### DISCUSSION TITLE
Saving Games Progress..

### DISCUSSION THREAD

**Discussion by Swapnekokitten** [skyline-emu/skyline#1601] [Ideas]
We would like to see skyline emulator adding autosaving progress done in games for example you update your skyline emulator for a new update and suddenly you went from the end back to where you started in the game this is a problem for all players and its annoying to see people suffering from this so please skyline save games progress

**Comment by QuackingCanary** [comment]
Hello, thanks for your interest in Skyline. Since a few weeks you can update the emulator without uninstalling the previous build. For some reason, if you have trouble with installing a new version, you can manually backup your saves located in `Android/data/skyline.emu`

**Reply by Swapnekokitten** [reply]
Whenever I wanna update skylibe emulator I have to Uninstall the previous build since when downloading a duplicate results in the phone saying you cannot download an application that is already installed

**Reply by QuackingCanary** [reply]
This shouldn't be the case. I just tested and indeed there is a problem. For now you'll have to wait for a fix and manually backup your saves
